extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/Base64String.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkus.vault.runtime;
+
+import static io.quarkus.vault.runtime.StringHelper.bytesToString;
+import static io.quarkus.vault.runtime.StringHelper.stringToBytes;
+import static java.util.Base64.getDecoder;
+import static java.util.Base64.getEncoder;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize(using = Base64StringSerializer.class)
+@JsonDeserialize(using = Base64StringDeserializer.class)
```

Comment:
I tried but could not get it to work so far.
I think the issue is that I instantiate the `ObjectMapper` directly into the `OkHttpVaultClient`, so I suppose the `JacksonModuleBuildItem` has no effect on it.
And I instantiate it directly because I use it from the vault config source, where I cannot use extensions such as jackson. It is one of those things you had noted on the initial review of the vault extension and led to create https://github.com/quarkusio/quarkus/issues/4162
I am not sure we can do better at this point, without full support for extensions fro within config sources.